data/mods/DinoMod/hints.json

```
@@ -10,6 +10,7 @@
       "I hear the zombies have been in the swamps.  Bad news if they bite a dinosaur before it bites them.",
       "I know there aren't alligators in the sewer, but I heard there was some kind of big lizard down there.  Probably not a good idea to check.",
       "Some of those big dinosaurs seem halfway all right.  I bet if you fed them something nice and gave them a pet you could ride them like a pony.  Or maybe they'd eat you instead.",
+      "I saw something with a large fin in the water. It was the size of a whale and gave me a menacing feeling. I'm gonna think twice before going back there.",
```

```suggestion
      "I saw something with a large fin in the water.  It was the size of a whale and gave me a menacing feeling.  I'm gonna think twice before going back there.",
```
Fixes the error in travis.

---

Travis still rejecting after commit